src/component/AddressForm/AddressLabel.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react";
```

Comment:
Not entirely sure why this is still in the repository. We are no longer using `label` and `input` to get our destination and origin locations ( see PR #4 )

Reply:
Yea I messed up and accidentally added them

Reply:
That file was meant to be for a different branch where I refactored the labels to be their own components, but I guess we're not using them anymore.